docs/generated/openapi.yaml

```diff
@@ -19,6 +19,62 @@ paths:
           $ref: '#/components/responses/InternalServerError'
       operationId: get-global-insight
       description: Returns a Global Insight object
+  /meshes/{mesh}/{policyType}/{policyName}/-resources/dataplanes:
```

Comment:
Why `-resources` is it something specific?

Reply:
Prefix by dash avoids any possible collision with dynamically generated urls from resource paths in the future.
It makes it very clear that this is not an actual entity type

Reply:
Is this a common open API thing? It seems like we'd be unlikely to dynamically generate a path `../resources` and if we do, isn't that just kind of confusing in spite of the `/-` prefix?

Reply:
It's not anything official in the openAPI spec for sure. But it's common in apis to have a syntax to clearly indicate that things are not a persisted but a computed resource. For example elastic search has a `_search` I think it's quite nice to have APIs that make this difference clear.

Reply:
Ok moved to _resources